pkg/master/master_test.go

```diff
@@ -900,7 +895,7 @@ func testInstallThirdPartyResourceRemove(t *testing.T, version string) {
 		etcdtest.AddPrefix("/ThirdPartyResourceData/company.com/foos/default/bar"),
 	}
 	for _, key := range expectedDeletedKeys {
-		_, err := client.Get(key, false, false)
+		_, err := etcdserver.Client.Get(key, false, false)
```

Comment:
can't you switch this to storage interface as well here?  Just 1 less thing to switch on client shift.